### PR TITLE
feat: added badge component

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added badge component to library.
badge types: status badge, text badge, Icon badge.
CPT2202031841-1512x698

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  display: flex;
+  align-items: center;
+}
+.badges-avatar img {
+  margin-right: 4rem;
+}
+.lg-badge-container {
+  position: relative;
+}
+.lg-badge-container .badgr-lg {
```
Comment (SanjayJatti) [inline components/badge.css:27]: I think it should be .badge-lg (spelling mistake)

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  top: -1rem;
+}
+.left-txt-bdg-container {
+  width: fit-content;
+  position: relative;
+  margin-top: 2rem;
+}
+.left-bdg {
+  left: 0;
+  top: 0rem;
```
Comment (nisargpawade) [inline components/badge.css:95]: the text badges are not aligned with the text

File: components/badge.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
+:root {
+}
+.badge-section {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+.badge-section h1 {
+  font-size: 48px;
```
Comment (SanjayJatti) [inline components/badge.css:9]: Give font size in rem. or u can create utility class for headings.

**Review by SanjayJatti — COMMENTED** [review]
Otherwise all looking fine.

File: components/badge.css (hunk 1 above)
Comment (abhijeetekad) [reply components/badge.css:27]: No, it's correct but I will write a more specific classname.

**Pull request merged**